configure.ac

```diff
@@ -212,6 +212,31 @@ AC_CHECK_FUNCS([ \
 	strsignal \
 ])
 
+dnl Various bfd section macros and functions like bfd_section_size() has been
+dnl modified starting binutils >= 2.34.
+dnl Check if the prototype is "bfd_section_size (sec)" or "bfd_section_size(bfd, ptr)"
+if test "$binutils_build_dir" != "NONE"; then
+    CFLAGS="-I$binutils_include_dir -I$bfd_include_dir $CFLAGS"
+fi
+
+AC_TRY_COMPILE([#include <bfd.h>],
+ [const asection *sec; bfd_section_size(sec);],
+ bfd_section_size_macro_has_one_arg=yes,
```

Comment:
name this more generically like `bfd_section_api_takes_bfd` or something.  they converted all the APIs at once, so there's no need to test each one individually.